Resources/ConfigPresets/WizardsDen/vulture.toml

```diff
@@ -1,7 +1,14 @@
 ﻿# Configuration preset used on Wizard's Den Leviathan
 
 [game]
-hostname = "[EN] Wizard's Den Vulture [US East 2]"
+desc = "Official English Space Station 14 servers. Vanilla, roleplay ruleset. Pre-release test branch.\nThis server is running the development branch. You can see upcoming changes ahead of the next bi-weekly release, but they may not be final.\nFeatures and changes on this server may be rolled back before release.\nPlease report any bugs you find, via github or Discord. Thank you!"
```

Comment:
"Official English testing server for Space Station 14.  Vanilla, roleplay ruleset. You can play with the newest changes here, but changes may not be final or stable, and may be reverted before the next stable release. Please report bugs to our Github, or via our community Discord."